utilities/kueue_utils.py

```
@@ -295,3 +420,157 @@ def wait_for_kueue_crds_available(client: DynamicClient) -> bool:
+        LOGGER.warning("kueue-controller-manager deployment not found, skipping pause")
+        return 0
+
+    replicas = controller.instance.spec.replicas
+    original_replicas = 1 if replicas is None else replicas
+    LOGGER.info(f"Pausing kueue-controller-manager (scaling from {original_replicas} to 0)")
+    ResourceEditor(patches={controller: {"spec": {"replicas": 0}}}).update()
+    _wait_for_kueue_controller_pods_gone(admin_client=admin_client)
+
+    try:
+        remove_kueue_visibility_api_services(admin_client=admin_client)
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)